sources/engine/Stride.Graphics/Vulkan/CommandList.Vulkan.cs

```diff
@@ -995,18 +995,13 @@ public void CopyMultisample(Texture sourceMultisampleTexture, int sourceSubResou
 
         public unsafe void CopyRegion(GraphicsResource source, int sourceSubresource, ResourceRegion? sourceRegion, GraphicsResource destination, int destinationSubResource, int dstX = 0, int dstY = 0, int dstZ = 0)
         {
-            // TODO VULKAN: One copy per mip level
```

Comment:
Is this todo outdated ?

Reply:
In general, several parts about the Vulkan implementation are not fully fleshed out, incomplete, or even unimplemented. But I'm not very knowledgeable about Vulkan, and also didn't read fully into the Vulkan files as this was just to replace `CopyMemory`. Maybe someone more versed with Vulkan...

Later down the line I'm planning to check the D3D 11 & 12 parts of the graphics stack. If there's something similar there I could return to check this.

Reply:
Sure, could you keep the comment in then, that way someone else can validate it ?

Reply:
Sorry, I forgot this comment. I'll copy the comment to the new method